src/Orleans/Messaging/RequestInvocationHistory.cs

```diff
@@ -2,24 +2,34 @@
 
 namespace Orleans.Runtime
 {
+    [Serializable]
+    internal class RequestInvocationInfo
```

Comment:
RequestInvocationHistory contains unnecessary ```GrainId``` and ```DebugContext``` properties, and the latter one may consume pretty large partof serialized message size.